draft-ietf-tsvwg-multipath-dccp.mkd

```diff
@@ -693,9 +693,9 @@ SubFlow).
 This permits break-before-make handover between
 SubFlows.  
 
-An MP-DCCP-level
-"Reset" allows the abrupt closure of the MP-DCCP connection
-using the MP_FAST_CLOSE suboption.
+It is necessary to provide an MP-DCCP-level
```

Comment:
"It is necessary" sounds a bit strange to me without an explanation why it is necessary.

Reply:
Sentence changed, 'necessary' removed.